tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -319,7 +319,10 @@ kbs_k8s_svc_host() {
 			-o jsonpath='{.spec.rules[0].host}' 2>/dev/null
 	elif kubectl get svc kbs-nodeport -n "$KBS_NS" &>/dev/null; then
 			local host
-			host=$(kubectl get nodes -o jsonpath='{.items[0].status.addresses[?(@.type=="ExternalIP")].address}' -n "$KBS_NS")
+			host=$(kubectl get nodes -o jsonpath='{.items[0].status.addresses[0].address}')
+			# I am not sure why we'd use the External / Internal IP as shown below,
```

Comment:
IIRC this was need to proper work with kcli. @ldoktor should know why for sure

Reply:
On some versions (kcli cluster) the externalIP/address was not set while InternalIP was. So yes, this is necessary. Thanks for keeping it.